app/routers/applications.py

```diff
@@ -240,6 +240,12 @@ async def verify_data_field(
         # Update a specific field in the application's `secop_data_verification` attribute.
         payload_dict = {key: value for key, value in payload.dict().items() if value is not None}
         key, value = next(iter(payload_dict.items()), (None, None))
+        if not key:
+            logger.error("Trying to update an empty key")
+            raise HTTPException(
```

Comment:
If a PUT request contains no data, I think it's okay to just return 200. As above, it shouldn't be possible in the frontend to construct such a request.

Reply:
I think just return the application, without creating an application action.